test/e2e/framework/framework.go.orig

```diff
@@ -0,0 +1,896 @@
+/*
```

Comment:
should this file be included?

Reply:
Urgh. No, thanks for pointing it out. I'll remove it.

Reply:
Removed and rebased...